maptivist/src/main/webapp/index.html

```diff
@@ -25,18 +24,21 @@
         </h1></a></li>
         <li style = "float:right"><a class = "navText" href="#home"><h1>explore</h1></a></li>
     </ul>
+    <p><a href = "#" id = "authLink">Sign in here</a></p>
     <div class = "title">
         <p class="mainText">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Turpis egestas integer eget aliquet nibh. Posuere morbi leo urna molestie at elementum. Tristique risus nec feugiat in. Aenean vel elit scelerisque mauris pellentesque pulvinar pellentesque habitant morbi. Eleifend donec pretium vulputate sapien nec sagittis aliquam malesuada bibendum. Neque sodales ut etiam sit amet nisl. Sit amet dictum sit amet justo donec. Praesent semper feugiat nibh sed pulvinar. Adipiscing elit pellentesque habitant morbi tristique senectus et netus. Odio ut sem nulla pharetra diam sit amet. Nec feugiat nisl pretium fusce id velit. Eros in cursus turpis massa tincidunt dui.
+            The problem with becoming an activist is that it is dangerous, because of this less people can get involved with the movement. In addition to the danger, the fact that we are in the midst of a pandemic also factors into why most people can’t go out and protest. Maptivist aims to remove any problems like this, and become sort of a “middle man” in terms of activism. In the age where everything is digitalized, it only makes sense for activism to be digitized as well. 
+            <br>
+            <br>
+            Especially in the current state of the United States, it is clear that people are looking to engage in organized activism. People have and continue to protest in the streets to make their voices heard. However, there are many groups of people who are not able to physically participate in local movements. Our project, Maptivist, aims to virtually connect our users to the physical places of social change. Mapvist is an interactive map web application to mark the locations of local activism. The web application will allow verified users to create and place markers on a Google Map to represent the location of a protest, meetings or important events about local issues. With the posted markers, all users will be able to search for markers based on specific issues, areas, or time periods. Maptivist provides users with a more inclusive path for activism, yet roots the users to the local places where change is occurring. 
```

Comment:
It's my error from the design doc but this line should be corrected:
"The web application will allow verified users to create and place markers on a Google Map to represent the location[s] of protest[s], meetings or important events about local issues."